components/infobox/wikis/overwatch/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,170 @@
+---
+-- @Liquipedia
+-- wiki=overwatch
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local HeroIcon = require('Module:HeroIcon')
+local Player = require('Module:Infobox/Person')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local GameAppearances = require('Module:GetGameAppearances')
+local TeamHistoryAuto = require('Module:TeamHistoryAuto')
+local Variables = require('Module:Variables')
+local Template = require('Module:Template')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+
+local _ROLES = {
+	-- Players
+	['dps'] = {category = 'DPS Players', variable = 'DPS', isplayer = true},
+	['flex'] = {category = 'Flex Players', variable = 'Flex', isplayer = true},
+	['support'] = {category = 'Support Players', variable = 'Support', isplayer = true},
+	['igl'] = {category = 'In-game leaders', variable = 'In-game leader', isplayer = true},
+	['tank'] = {category = 'Tank Players', variable = 'Tank', isplayer = true},
+
+	-- Staff and Talents
+	['analyst'] = {category = 'Analysts', variable = 'Analyst', isplayer = false},
+	['observer'] = {category = 'Observers', variable = 'Observer', isplayer = false},
+	['host'] = {category = 'Hosts', variable = 'Host', isplayer = false},
+	['journalist'] = {category = 'Journalists', variable = 'Journalist', isplayer = false},
+	['expert'] = {category = 'Experts', variable = 'Expert', isplayer = false},
+	['coach'] = {category = 'Coaches', variable = 'Coach', isplayer = false},
+	['caster'] = {category = 'Casters', variable = 'Caster', isplayer = false},
+	['talent'] = {category = 'Talents', variable = 'Talent', isplayer = false},
+	['manager'] = {category = 'Managers', variable = 'Manager', isplayer = false},
+	['producer'] = {category = 'Producers', variable = 'Producer', isplayer = false},
+	['admin'] = {category = 'Admins', variable = 'Admin', isplayer = false},
+}
+_ROLES['assistant coach'] = _ROLES.coach
```

Comment:
Is this used? If not, I recommend leaving it out

Reply:
In terms of producers or admins.. not really. 
In terms of assistant coaches, yes, quite a lot
![image](https://user-images.githubusercontent.com/58013431/177418965-3ecc7b7d-f7d7-4a51-b28b-f60d128b856f.png)

Reply:
Is it used in the Player Infobox though?

Reply:
Which part? Assistant Coaches or the Producer/Admin? 

Admin they dont use at all, producer they code as a caster in the infobox, and assistant coach im not seeing used correctly anywhere (story of the wiki)

Reply:
Assistant Coach

Reply:
All of the assistant coaches i have looked at are hard coded to coach under their role, and have a manually inputted line in their bio specifying them as the assistant coach